### PR TITLE
Impl OperationModifier for actix_multipart

### PR DESCRIPTION
**Pull request by sfisol**
I'm not sure if these types covers all possible multipart cases, but better this than nothing :-)

### CODE REVIEW

File: core/Cargo.toml [hunk 1]
```diff
@@ -28,7 +29,7 @@ serde_yaml = "0.8"
 uuid = { version = "0", optional = true }
 
 [features]
-actix = ["v2", "actix-http", "actix-web"]
+actix = ["v2", "actix-http", "actix-web", "actix-multipart"]
```
Comment (wafflespeanut) [inline core/Cargo.toml:32]: Let's have a `multipart` feature instead, which depends on `actix-multipart` please? Like, how we have it for `chrono` and `datetime` (the dependency should be opt-in).

**Comment by wafflespeanut** [comment]
Closed and reopened so that Travis picks it up.

File: core/Cargo.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ repository = "https://github.com/wafflespeanut/paperclip"
 [dependencies]
 actix-web = { version = "2.0", optional = true }
 actix-http = { version = "1.0", optional = true }
+actix-multipart = { version = "0.2", optional = true }
```
Comment (wafflespeanut) [inline core/Cargo.toml:15]: Does this work at the moment? There's no feature in `core` that ties this dependency.

Comment (sfisol) [reply core/Cargo.toml:15]: It seems that if there is no feature then plain optional dependency is activated. I haven't found any info in cargo documentation about this, but I've checked it and it works for me.

Comment (wafflespeanut) [reply core/Cargo.toml:15]: Interesting. I guess this is why they don't allow feature name to collide with dependency name. :smile:

**Pull request merged**